packages/db/src/schema.prisma

```diff
@@ -265,3 +267,24 @@ model Gloss {
 
   @@id([wordId, languageId])
 }
+
+/// The entry in a lexicon associated with a particular lemma.
+/// TODO: determine if we need a better name
+model LemmaLexiconEntry {
```

Comment:
I wasn't sure what to name it.

Reply:
I can think of two alternatives, but I don't have a strong preference
1. LexiconEntry - Lexicon already communicates that we are talking about dictionary like entries.
2. LemmaResource - Technically, Strongs isn't a lexicon, but a concordance. If we call these resources for lemmas, then a LemmaResource would have a resourceCode instead. Also, a Lemma would have resources rather than lexiconEntries